.gitignore

```diff
@@ -16,6 +16,7 @@
 # Exclude IDE and Editor files
 /.idea/
 *.sublime*
+.project
```

Comment:
This comment says the rfc is at text/0000-raw-txn-header.md but it actually been moved to text/0023-raw-txn-header.md Please either update this or remove that line and just point to the pr